BlueprintCore/BlueprintCore/Utils/EncyclopediaTool.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using BlueprintCore.Utils;
+using BlueprintCore.BlueprintCore.Internal;
+
+namespace BlueprintCore.BlueprintCore.Utils;
+public static class EncyclopediaTool
```

Comment:
Add a comment indicating this was modified from TTT & link to the repo.

Reply:
Done.